angular-chart.js

```diff
@@ -185,6 +192,8 @@
               }, 50, false);
             }
             if (! scope.data || ! scope.data.length) return;
+            // Disable the existing watch. It has been installed before and it is installed again
+            cancel_watch_for_colours();
```

Comment:
I'm not sure. I noticed the chart was drawn multiple times (including animation) and it bothered me. The change I created prevents that from happening on both chrome and IE. Output written to the Developer Tools console made me believe the resetChart() method is invoked after the first $watch finishes, drawing the chart again. I've seen this happening on both IE11 and Chrome, using angular 1.3 and 1.4